CHANGELOG.md

```diff
@@ -1,6 +1,32 @@
 Change Log
 ==========
 
+# Version 3.7.3
+
+_2022-12-20_
+
+This release contains a handful of bug fixes and improvements, and also discontinues the legacy JS artifacts. 
+
+Many thanks to @StefanChmielewski and @chao2zhang for contributing to the project! 🧡
+
+## ⚙️ JS legacy artifacts (#4591)
```

Comment:
Sorry missed that but we could be more explicit there for those like me that just scan the `h2` titles

```suggestion
## ⚙️ Removed JS legacy artifacts (#4591)
```